auth/src/main/java/com/firebase/ui/auth/AuthUI.java

```diff
@@ -304,8 +144,8 @@ public static AuthUI getInstance(FirebaseApp app) {
     }
 
     /**
-     * Default theme used by {@link SignInIntentBuilder#setTheme(int)} if no theme
-     * customization is required.
+     * Default theme used by {@link SignInIntentBuilder#setTheme(int)} if no theme customization is
```

Comment:
@samtstern Yeah, sorry. Intellij, just does that automatically when formatting stuff. (I should poke around and try to find where the setting is to disable it, but I've been lazy. I'll look into it at some point.)